### DISCUSSION TITLE
What models can benefit from future_regressor?

### DISCUSSION THREAD

**Discussion by mreismendes** [winedarksea/AutoTS#70] [Q&A]
Hello everyone!
I would like to know what models can benefit from `future_regressor`?

Thank you!

**Comment by winedarksea** [comment]
@mreismendes 
https://github.com/winedarksea/AutoTS/blob/master/extended_tutorial.md#models

See column 'Use Regressor'

caveat: all of these models can either use or not use the regressor, as of now there is no way to force it to **always** use the regressor. Open an issue if always using the regressor is something you think would be valuable.

**Reply by mreismendes** [reply]
@winedarksea Is there a specific reason why GluonTS isn't on the list? Models like DeepAR can benefit from them.

**Reply by winedarksea** [reply]
@mreismendes the reason is that I couldn't figure out how to make the regressors work with GluonTS when I tried a year ago. It's actually on my list of things to do, somewhere way near the bottom. If you have a good example of regressors with it, that would help!
